ArchiSteamFarm/Json/Steam.cs

```diff
@@ -476,16 +480,137 @@ internal sealed class InventoryResponse : NumberResponse {
 			private InventoryResponse() { }
 
 			internal sealed class Description {
+				[JsonExtensionData]
+#pragma warning disable 649
+				internal ImmutableDictionary<string, JToken> AdditionalProperties;
```

Comment:
Why this isn't `readonly`?

Reply:
It was highlighted as an error, but for some reason it is still compiling, I'll revert it

Reply:
Warnings are converted to errors in `Release` build, but this is what we have pragmas for.